### PR TITLE
Friends app - restyling

### PR DESCRIPTION
**Pull request by stoyanvi**

### CODE REVIEW

File: platform-friends-hybrid/styles/main.css [hunk 1]
```diff
@@ -1,18 +1,32 @@
 .cf {
-    zoom: 1;
+  zoom: 1;
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: Is this zoom really needed?

File: platform-friends-hybrid/styles/main.css [hunk 2]
```diff
@@ -32,585 +46,610 @@
...
+#friendsApp .hide-text {
+  text-indent: -999px;
+  overflow: hidden;
 }
 
 .disabled {
-    position: relative;
-    display: block;
+  position: relative;
+  display: block;
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: If we target WP8.1+, we can probably just use pointer-events: none;
http://caniuse.com/#search=pointer-events

File: platform-friends-hybrid/styles/main.css [hunk 3]
```diff
@@ -32,585 +46,610 @@
...
-#friendsApp.km-flat form .km-listinset > li:first-child,
-#friendsApp.km-flat form .km-listgroupinset .km-list > li:first-child {
-    border-width: 0 0 1px;
-    border-color: #eaeced;
-    -webkit-border-radius: 2px 2px 0 0;
-    -moz-border-radius: 2px 2px 0 0;
-    border-radius: 2px 2px 0 0;
+#friendsApp.km-flat form .km-list > li,
+#friendsApp .btn {
+  -webkit-border-radius: 0;
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: border-radius was unprefixed too long time ago :)
http://caniuse.com/#search=border-radius

File: platform-friends-hybrid/styles/main.css [hunk 4]
```diff
@@ -32,585 +46,610 @@
...
-    border-color: #eaeced;
-    background-color: #fff;
-    -webkit-box-shadow: none;
-    box-shadow: none;
-    color: #34495e;
+  padding: .75em 0 1em;
+  border-width: 0;
+  color: #4d4d4d;
+  background-color: #fff;
+  -webkit-box-shadow: none;
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: We can probably get away without prefix here too, unless we still support Android 2.3:
http://caniuse.com/#search=box-shadow
https://developer.android.com/about/dashboards/index.html - 6% only lately.

File: platform-friends-hybrid/styles/main.css [hunk 5]
```diff
@@ -32,585 +46,610 @@
...
-    padding: 0 1em 1em;
-    background: #fff none;
+#friendsApp .activity-view .picture {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  width: auto;
+  min-height: 100%;
+  -webkit-transform: translate(-50%, -50%);
+      -ms-transform: translate(-50%, -50%);
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: -ms-transform is unneeded for IE10+.

File: platform-friends-hybrid/styles/main.css [hunk 6]
```diff
@@ -32,585 +46,610 @@
...
+
+#friendsApp.km-flat,
+#friendsApp.km-flat .km-navbar,
+#friendsApp.km-flat .km-content,
+input[type='date'],
+input[type='datetime-local'],
+input[type='month'],
+input[type='time'],
+input[type='week'] {
+  font-family: 'Roboto', sans-serif;
```
Comment (bundyo) [inline platform-friends-hybrid/styles/main.css]: Not sure if this is needed - the original Flat skin would probably use Roboto Light if available anyway (unless we want the normal one specifically):
https://github.com/telerik/kendo-ui-core/blob/master/styles/mobile/flat/base.less#L9

**Comment by bundyo** [comment]
:+1:

File: platform-friends-hybrid/styles/main.css (hunk 6 above)
Comment (stoyanvi) [inline platform-friends-hybrid/styles/main.css]: We want Roboto specifically.

**Comment by AntonDobrev** [comment]
:+1:

**Pull request merged**